app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About | Ruby on Rails Tutorial Sample App</title>
+  </head>
+  <body>
+    <h1>About</h1>
+    <p>
+      The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
+      Tutorial</em></a> is a
+      <a href="http://www.railstutorial.org/book">book</a> and
+      <a href="http://screencasts.railstutorial.org/">screencast series</a>
+      to teach web development with
+      <a href="http://rubyonrails.org/">Ruby on Rails</a>.
+      This is the sample application for the tutorial.
+    </p>
+  </body>
```

Comment:
Sao mỗi file [application.html.erb] là đã dùng I18n vậy
Sử dụng I18n cho toàn bộ project chứ. Tiêu biểu là html.ẻb